src/clients/multiplayerClient.js

```diff
@@ -0,0 +1,29 @@
+import { io } from 'socket.io-client';
+
+export default class MultiplayerClient {
+  client
+  ENDPOINT = 'localhost:3000'
```

Comment:
Yes, please add at least a TODO so we don't forget to add a constant or setting value